ui/imports/shared/controls/chat/ProfileHeader.qml

```diff
@@ -151,10 +151,10 @@ Item {
                     if (!!root.icon)
                         Global.openMenu(editImageMenuComponent, this)
                     else
-                        Global.openChangeProfilePicPopup(tempIcon);
+                        Global.openChangeProfilePicPopup(setTempIcon);
```

Comment:
Just a rename because the code was confusing (I was searching for the call back and turns out `tempIcon` is a function)